whiteblock/cmd/transactions.go

```diff
@@ -86,6 +105,33 @@ Optional Parameters:
 	},
 }
 
+var sendToTxCmd = &cobra.Command{
+	// Hidden: true,
+	Use: "to",
+	Short: "Send transaction data to an account",
+	Long: `
+The user must specify the flags that will be used for sending transaction data.
+Send a transaction data to an account.
+
+Required Parameters: 
+	--destination <address> --value <amount> --data <transaction data>
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		command := ""
+		params := []string{}
+
+		if !(len(toFlag) > 0) || valueFlag == 0 || !(len(dataFlag) > 0)  {
+			util.Print("Required flags were not provided. Please input the required flags.")
```

Comment:
Use `util.PrintStringError` or `util.PrintErrorFatal` instead

Reply:
Currently, that will exit with a success status code and the user may not know what is wrong